deployments/deploy-ssh.sh

```diff
@@ -251,6 +251,16 @@ done
 
 echo "checking if we need to override test suite: ${TEST_SUITE}"
 
+# TODO: Remove this after all longevity jobs switch to 'bin/longevity.test' for TEST_SUITE.
+case $FOCUS_TESTS in
+  Longevity|UpgradeLongevity|BackupLongevity)
```

Comment:
Could we have a wildcard entry *Longevity* so that even if any new test is added TEST_SUITE is set to `bin/longevity.test` for now.

Reply:
I thought so, but if someone includes the 'Longevity' keyword in any of their tests, it will incorrectly set the TEST_SUITE.

Reply:
Shouldn't the `TEST_SUITE` parameter be passed from the pipeline?

Reply:
It is a parameter in all the pipelines but it is difficult to list all longevity pipelines as of now.